sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -45,6 +45,8 @@ export enum AppRoutes {
   SearchInstitutions = "search-institutions",
   InstitutionDetail = "institution-detail/:institutionId",
   Profile = "profile",
+  PendingDesignations = "institution/designation/pending",
+  DesignationAESTView = "institution/designation/view/:designationId/institution/:institutionId?",
```

Comment:
`view` is not needed just `"institution/designation/:designationId/institution/:institutionId?"` is fine, IMO